README.md

```diff
@@ -75,7 +75,7 @@ You can then [manually run a workflow](https://docs.github.com/en/actions/managi
 
 ## Without sudo
 
-By default we run the commands using sudo. If you get `sudo: not found` you can use the parameter below to execute the commands directly.
+By default we run install commands using sudo on Linux. If you get `sudo: not found` you can use the parameter below to execute the commands directly.
```

Comment:
It's not really _commands_ we install, but _packages_, right?

Reply:
It was an English mistake.

```suggestion
By default we run installation commands using sudo on Linux. If you get `sudo: not found` you can use the parameter below to execute the commands directly.
```

`installation commands` are intended to `apt-get update` `apt-get install -y openssh-client xz-utils` etc.